CHANGES.rst

```diff
@@ -2854,6 +2862,9 @@ Bug Fixes
   - ``treat_deprecations_as_exceptions`` has been fixed to recognize Astropy
     deprecation warnings. [#3015]
 
+  - Converted representation of progress bar units without suffix
+    from float to int in console.human_file_size. [#2201,#2202,#2721,#3299]
```

Comment:
Do there need to be spaces after the commas in the version number for the link-maker thingie to work right? (Not sure...)